### ISSUE TITLE
Trainin result shows that val losses are 0

### ISSUE THREAD

**Issue by darkstar1227** [WongKinYiu/yolov9#264]
![image](https://github.com/WongKinYiu/yolov9/assets/83396614/43f38335-5537-4268-b1ae-fcbe4365a46a)
![image](https://github.com/WongKinYiu/yolov9/assets/83396614/ba5041ea-d47e-44c9-978c-7468b9c844c7)
- I am using Yolov8 and Yolov9 to do the comparation. Using the same datasets.
- Yolov8 has a normal result curve as Fig. 1.
- Yolov9 gave me that definitely wrong result curve as Fig. 2.
- I am using A100 to train it.
- Cuda 11.3, torch 1.11.0, pillow 9.5
- I have changed the pillow version because it shows a size error. I used the solution of yolov5.

**Comment by Youho99** [comment]
Check #132
Val Losses are not yet implemented for Yolov9 models
Use val_dual.py instead

**Comment by Visiblemiracle** [comment]
try to reduce epochs number

**Comment by jasfa** [comment]
I trained yolov9-c with train_dual-py, where val_dual-py was used, and val/loss was still 0

**Comment by Mr-Liiiii** [comment]
@WongKinYiu  是用val_dual.py 应该也是0吧

**Comment by babakbch** [comment]
same problem here :(
